### PR TITLE
Scroll to current envelope in thread

### LINKED ISSUE #3677: Scroll to "current" message in thread
### Feature Request

When you click the envelope of a new message in a long thread, you'll see a list of the old message first and need to scroll to see the message you just wanted to see.

Bildschirmfoto von 2020-09-29 10-54-49

It would be nicer if we could initialize the scrolling position so the *current* message's header is shown on top and you see the previous messages only if you scroll.

#### Summary

Scroll to the current message of a thread.

cc @GretaD

### PR DESCRIPTION
**Pull request by st3iny**
Fixes #3677

Scroll to the relevant envelope in a thread.

### CODE REVIEW

**Comment by st3iny** [comment]
My arch nemesis the coverage at it again :cloud_with_lightning:

File: src/components/ThreadEnvelope.vue [hunk 1]
```diff
@@ -197,9 +197,10 @@ export default {
 			}
 		},
 	},
-	mounted() {
+	async mounted() {
 		if (this.expanded) {
-			this.fetchMessage()
+			await this.fetchMessage()
+			this.$nextTick(() => this.scrollTo())
```
Comment (ChristophWurst) [inline src/components/ThreadEnvelope.vue:203]: scroll to *what*? :P

how about `scrollToCurrentEnvelope`?

Comment (st3iny) [reply src/components/ThreadEnvelope.vue]: Agreed

**Comment by st3iny** [comment]
On second thought, we could remove the smooth scrolling effect. It certainly looks nice but I'm actually not a big fan of animations. Opinions?

**Review by ChristophWurst — APPROVED** [review]
Code looks good, didn't test

**Comment by StCyr** [comment]
> 
> 
> On second thought, we could remove the smooth scrolling effect. It certainly looks nice but I'm actually not a big fan of animations. Opinions?

I agree, I've tested and I prefer the 'auto' behavior'. Could you change it?

**Comment by st3iny** [comment]
Removed smooth scrolling and rebased the branch.

**Pull request merged**